src/resources/schedule.yml

```diff
@@ -1,53 +1,53 @@
 schedule:
```

Comment:
Are we sure about these dates?  If not, it might be best to avoid pushing them, so it's clear they need to be checked.

Reply:
As far as I understand, we're not, and this is also irrelevant to the pull, so I would suggest we don't merge this in this pull

Reply:
#358